core/src/lib.rs

```diff
@@ -3,6 +3,8 @@
 //! Clementine Core is the backbone of Clementine. As the name suggests,
 //! Clementine Core provides core functionalities for Clementine to operate.
 
+#![allow(clippy::too_many_arguments)]
```

Comment:
This shouldn't be global. We should fix that functions or add this to start of the functions (only affects that function) and leave a TODO note.

Reply:
Fixed with [0f731ac](https://github.com/chainwayxyz/clementine/pull/240/commits/0f731aca2c332556beb02592954dbf855f81f1c3).